mmv1/third_party/terraform/services/dataplex/resource_dataplex_glossary_category_test.go.tmpl

```diff
@@ -0,0 +1,92 @@
+package dataplex_test
+
+{{- if ne $.TargetVersionName "ga" }}
```

Comment:
Remove this and the matching `{{- end }}` at the bottom since these resources support GA.